CMakeLists.txt

```diff
@@ -5,11 +5,13 @@ set (CPACK_PACKAGE_VERSION_MAJOR "15")
 set (CPACK_PACKAGE_VERSION_MINOR "0")
 set (CPACK_PACKAGE_VERSION_PATCH "0")
 if (DEFINED GIT_COMMIT)
-set (CPACK_PACKAGE_VERSION_PATCH "GIT-${GIT_COMMIT}")
+	set (CPACK_PACKAGE_VERSION_PATCH "GIT-${GIT_COMMIT}")
 endif (DEFINED GIT_COMMIT)
 set (CPACK_PACKAGE_VENDOR "Nano Currency")
 
 set(CMAKE_INSTALL_RPATH "@executable_path/../Frameworks")
+# Create all libraries and executables in the root binary dir
+set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR})
```

Comment:
Isn't having output directories for Debug/Release better than polluting the root dir ? An output directory makes it a tad easier/safer to wipe/move/copy build artifacts IMO. Is there an advantage?

Reply:
This was needed to fix the tests, which expect rai_node to be in CMAKE_BINARY_DIR. I thought of building the install target and running the tests from the install/bin folder but the install target does not include the tests, so half of the executables come from there and half from the build folder. Instead of adding the tests to the install target I found this to be simpler way to keep the path of the executables as in master.

For any real distribution the install target is more appropriate but for the tests I am not sure.